assets/UIExtensionArchitecture.svg

```diff
@@ -15,7 +15,7 @@
    version="1.1"
    inkscape:version="0.92.4 (5da689c313, 2019-01-14)"
    sodipodi:docname="UIExtensionArchitecture.svg"
-   inkscape:export-filename="D:\TFS_Data\Ringtail\ringtail-extension-sdk\assets\UIExtensionArchitecture.png"
+   inkscape:export-filename="D:\TFS_Data\git\discover-extension-sdk\assets\UIExtensionArchitecture.png"
```

Comment:
Regardless of "ringtail" v "Discover", it seems wrong to have a hard-coded path like this that's dependent on someone's git install directory... but I don't really get what this file is for.  Maybe it doesn't matter.

Reply:
Fair enough.  After reading (surface level) about this, it is something relative to the software (Inkscape - a drawing program) that automatically generates this.  There are some discussions I see around a bug fix for that software.   At any rate, I will just remove this as it will really only affect an end-user of Inkscape that might actually try and pull this svg into that program directly.